test-app/package.json

```diff
@@ -0,0 +1,111 @@
+{
+  "name": "test-app",
+  "version": "2.0.0-beta.3",
+  "private": true,
+  "description": "The test app for ember-cli-page-object",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/san650/ember-cli-page-object.git",
+    "directory": "test-app"
+  },
+  "license": "MIT",
+  "author": "Santiago Ferreira",
+  "directories": {
+    "doc": "doc",
+    "test": "tests"
+  },
+  "scripts": {
+    "build": "ember build --environment=production",
+    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel \"lint:!(fix)\"",
+    "lint:fix": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*:fix",
+    "lint:hbs": "ember-template-lint .",
+    "lint:hbs:fix": "ember-template-lint . --fix",
+    "lint:js": "eslint . --cache",
+    "lint:js:fix": "eslint . --fix",
+    "start": "ember serve",
+    "prepack": "ember ts:precompile",
+    "postpack": "ember ts:clean",
+    "test": "npm-run-all lint test:*",
+    "test:ember": "ember test"
+  },
+  "devDependencies": {
+    "@babel/eslint-parser": "^7.17.0",
+    "@babel/plugin-proposal-decorators": "^7.17.9",
+    "@ember/optional-features": "^2.0.0",
+    "@ember/test-helpers": "^2.6.2",
+    "@embroider/test-setup": "^1.6.0",
+    "@glimmer/component": "^1.1.2",
+    "@glimmer/tracking": "^1.1.2",
+    "@types/ember": "^3.1.1",
+    "@types/ember-qunit": "^3.4.13",
+    "@types/ember-resolver": "^5.0.10",
+    "@types/ember__application": "^3.16.2",
+    "@types/ember__array": "^3.16.4",
+    "@types/ember__component": "^3.16.4",
+    "@types/ember__controller": "^3.16.4",
+    "@types/ember__debug": "^3.16.3",
+    "@types/ember__engine": "^3.16.2",
+    "@types/ember__error": "^3.16.1",
+    "@types/ember__object": "^3.12.5",
+    "@types/ember__polyfills": "^3.12.1",
+    "@types/ember__routing": "^3.16.12",
+    "@types/ember__runloop": "^3.16.3",
+    "@types/ember__service": "^3.16.1",
+    "@types/ember__string": "^3.16.3",
+    "@types/ember__template": "^3.16.1",
+    "@types/ember__test": "^3.16.1",
+    "@types/ember__test-helpers": "^1.7.3",
+    "@types/ember__utils": "^3.16.2",
+    "@types/htmlbars-inline-precompile": "^1.0.1",
+    "@types/jquery": "^3.3.31",
+    "@types/qunit": "^2.11.1",
+    "@types/rsvp": "^4.0.3",
+    "broccoli-asset-rev": "^3.0.0",
+    "coveralls": "^3.0.0",
+    "ember-auto-import": "^2.4.1",
+    "ember-cli": "~4.0.0",
+    "ember-cli-babel": "^7.26.11",
+    "ember-cli-code-coverage": "1.0.0-beta.3",
+    "ember-cli-dependency-checker": "^3.3.1",
+    "ember-cli-doc-server": "1.1.0",
+    "ember-cli-htmlbars": "^6.0.1",
+    "ember-cli-inject-live-reload": "^1.8.2",
+    "ember-cli-page-object": "2.0.0-beta.3",
```

Comment:
I've just noticed this, and I wonder if this is supposed to be a relative path, or use `*` instead of the hardcoded version?